storage/innobase/dict/dict0dd.cc

```diff
@@ -6362,13 +6349,8 @@ const char *dd_process_dd_tables_rec_and_mtr_commit(
 
   /* Get the se_private_id field. */
   field = (const byte *)rec_get_nth_field(
-<<<<<<< HEAD
-      rec, offsets, dd_tables->field_number("se_private_id") + DD_FIELD_OFFSET,
-=======
       nullptr, rec, offsets,
-      dd_object_table.field_number("FIELD_SE_PRIVATE_ID") + DD_FIELD_OFFSET,
->>>>>>> mysql-8.0.29
```

Comment:
Do you know why we changed from "FIELD_SE_PRIVATE_ID" to se_private_id?

Reply:
@satya-bodapati We/upstream did not change.
PXB gather the information as `dd_tables->field_number("se_private_id")`
Upstream create a `dd_object_table` - https://github.com/mysql/mysql-server/blob/mysql-8.0.29/storage/innobase/dict/dict0dd.cc#L5332 and uses that object to fetch the field number via `dd_object_table.field_number("FIELD_SE_PRIVATE_ID")` . From what I can see, we first added the optimization of fetching the field number as name rather than fixed number. Later upstream did the same in a different way but we never conciliated to their way. 
However, here what changed was the first parameter `nullptr` we pass to `rec_get_nth_field` as this is new work added as part of INSTANT ADD/DROP column commit.